e2e/scripts/actions/exec.test

```diff
@@ -0,0 +1,54 @@
+#!/singularity/test
+# exec tests
+
+source common.func
+
+# workdir
```

Comment:
Creating these temporary directories looks like they could be another built-in that does a bit more of this work for you.

Reply:
`mktmpdir/mktmpfile` are already function wrappers around `create-tmpdir/create-tmpfile` builtins